security/security-spring/pom.xml

```diff
@@ -19,9 +19,15 @@
     <dependencies>
       <dependency>
         <groupId>org.mskcc.cbio</groupId>
-        <artifactId>persistence-mybatis</artifactId>
```

Comment:
@khzhu I believe this is a change from when we decoupled the security and persistence module... this change might go away actually when we rebase this pr to the latest `rc`

Reply:
The dependency was changed because of the class org.cbioportal.service.util.JwtUtils ... we wrote this class to provide services for creating and validating JWT access tokens. In the security-spring module there is a Filter class called TokenAuthenticationFilter.java which is responsible for intercepting requests to the web API and finding an attached Data Access Token in a request header. If the token is present, the filter will validate the token before allowing the request to be processed by the web layer (API Controllers). The filter class can accept either UUID tokens or JWT tokens, and the validation of JWT tokens required the functions in JwtUtils.

We debated whether we should locate this JwtUtil class in a separate module, so that we would not need to import the entire service layer here, but decided against creating a new maven module (we have quite a few already). So here we import the service module as a dependency, and since that module is dependent on the persistence-mybatis module, we also get the functionality of the lower layers imported here as well inherently.

Reply:
well explained, thank you, @sheridancbio.